app/components/IconsPickerModal.js

```diff
@@ -0,0 +1,57 @@
+import React, { PropTypes } from 'react';
+import { View, ListView, TouchableOpacity } from 'react-native';
+import _ from 'lodash';
+import Modal from 'react-native-modal';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+import styles from '../styles/IconsPickerStyles';
+import icons from '../constants/accountIcons';
+
+const chunkedIconsList = _.chunk(icons, 4);
```

Comment:
you can replace it by similar function from Ramda (don't use different packages with similar functionality)